src/SummerShop.WebApi/Controllers/ProductController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using SummerShop.WebApi.Data;
+using SummerShop.WebApi.Models.Product;
+
+namespace SummerShop.WebApi.Controllers;
+
+[Route("API/[controller]")]
+[ApiController]
+public class ProductController : ControllerBase
+{
+    private readonly ShopDbContext _context;
+    // GET
+    [HttpGet]
+    public async Task<ActionResult<IEnumerable<Product>>> GetProducts()
+    {
+        return await _context.Products.ToListAsync();
+    }
+    
+    [HttpGet("{id}")]
+    public async Task<ActionResult<Product>> GetArticle(int id)
+    {
+        var product = await _context.Products.FindAsync(id);
+
+        if (product == null)
+        {
+            return NotFound();
+        }
+
+        return product;
```

Comment:
Can you wrap this in an Ok object result?

Reply:
Okkkk